src/bsonjs.c

```diff
@@ -85,15 +98,18 @@ PyDoc_STRVAR(dump__doc__,
 "This function wraps `bson_as_json` from libbson.");
 
 static PyObject *
-dump(PyObject *self, PyObject *args)
+dump(PyObject *self, PyObject *args, PyObject *kwargs)
 {
     PyObject *bson, *file, *json;
-
-    if (!PyArg_ParseTuple(args, "SO", &bson, &file)) {
+    static char *kwlist[] = {"", "", "extended", NULL};
```

Comment:
What do you think about renaming "extended" to "mode"?

Reply:
Done.